stdlib/public/Concurrency/Task.swift

```diff
@@ -695,9 +695,12 @@ extension Task where Success == Never, Failure == Never {
 /// Storing an unsafe reference doesn't affect the task's actual life cycle,
 /// and the behavior of accessing an unsafe task reference
 /// outside of the `withUnsafeCurrentTask(body:)` method's closure isn't defined.
-/// Instead, use the `task` property of `UnsafeCurrentTask`
-/// to access an instance of `Task` that you can store long-term
-/// and interact with outside of the closure body.
+///
+/// If necessary to store a `Task` object itself, it is possible to create
+/// an un-structured task and store it, however it is not possible to retrieve
+/// the "current" task as an object. Instead, interactions with the "current"
+/// task are performed with the `currentPriority` and similar APIs which safely
+/// query the current task, without exposing its actual identity.
```

Comment:
```suggestion
/// There's no safe way to retrieve a reference to the current task
/// and save it for long-term use.
/// To query the current task without saving a reference to it,
/// use properties like `currentPriority`.
/// If you need to store a reference to a task,
/// create an unstructured task using `Task.detached(priority:operation:)` instead.
```

Reply:
I'm not sure why this PR added a paragraph break — the discussion is still about long-term references to tasks.

Revised for typical style in reference, and to order the points by their expected relevance.

Reply:
Agree with the wording but I'll add a "no way to _safely_ ..." there because there is an unsafe (in swift wording) way to do it. Thanks for review @amartini51 !

Reply:
ah actually it does say that -- nevermind all good